source/sass/patterns/molecules/info-bar.scss

```diff
@@ -0,0 +1,61 @@
+@import "../../variables/color";
+@import "../../variables/font";
+@import "../../mixins/positioning";
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+$_info-bar-gutter: 8px;
+$_info-bar-image-offset: $font-size;
+$_info-bar-image-fudge-factor: 4px;
```

Comment:
👀

Reply:
Without flexbox support the images need bringing down slightly to visually centre them, an arbitrary value of 4px achieves this. It's not pretty, but putting it into a variable at least calls it what it is.

Reply:
Given the wide support for Flexbox, and that it's just a bit misaligned without, don't think we should try and maintain this (eg what happens if someone tweaks the font size?).

Reply:
With this hack, if someone changes the font size it might look broken then in IE11, but without this hack it looks broken now:
<img width="385" alt="Screenshot 2019-06-25 16 31 10" src="https://user-images.githubusercontent.com/2893480/60111864-aa3b9280-9766-11e9-838a-6ef2f1058e01.png">

Could relate the fudge factor to the font-size to help mitigate the effects of a font-size change, something like:
`$_info-bar-image-fudge-factor: #{$font-size / 4}px;`

Reply:
See 55caba4.

Reply:
I wouldn't describe that as broken, but as a compromise I've tried https://github.com/libero/pattern-library/pull/115/commits/e4a6e47616391d5bb5e5d02a86a9eab3396c765d, which fixes the image block size without Flexbox (to be 1 line).